_workinggroups/pyhep.md

```diff
@@ -1,14 +1,36 @@
 ---
-title: PyHEP Workshops
-author: "Eduardo Rodrigues"
-layout: default
+title: PyHEP - "Python in HEP"
+layout: plain
+redirect_from:
+  - /activities/pyhep.html
 ---
 
-# PyHEP "Python in HEP" Series of Workshops
+The PyHEP working group brings together a community of developers and users of Python in Particle Physics, with the aim of improving
+the sharing of knowledge and expertise. It embraces the broad community, from HEP to the Astroparticle and Intensity Frontier communities.
+
+The group is currently coordinated by Chris Tunnell (Rice U.), Eduardo Rodrigues (U. Cincinnati) and Graeme Stewart (CERN).
+All coordinators can be reached via <hsf-pyhep-organisation@googlegroups.com>.
+
+# Getting Involved
+
+Everyone is welcome to join the community and participate by means of the following:
+
+[Gitter channel PyHEP](https://gitter.im/HSF/PyHEP){:target="_pyhep_gitter_channel} for any informal exchanges.
```

Comment:
Wouldn't it be better to make this all a bulleted list? The formatting looks quite strange having this as just a pagaraph, following a colon above.

Reply:
Sure, will do.